### ISSUE TITLE
Main navigation not marked up as a landmark

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#287]
# What’s the issue?

The main navigation in the site header is not marked up as a navigation region. All other navigation regions on the site are marked up correctly.

# Why it matters

Landmark roles enable screen reader users to easily navigate to different regions of the page. Landmark roles should be applied consistently otherwise users may
inadvertently miss content.

# Our recommendation

Add `<nav role=”navigation” ...>` to the main navigation with a label of `main`.

Note that currently the section navigation on the content pages (e.g. About Us, Join our Team) located on the left hand side is labelled as `main`. The label should be changed to “section” or equivalent to distinguish from main.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-gov-au/pull/295.

Menus are now labelled in templates as 'Main', 'Section' and 'Footer', which will not change if the name of the block changes. If menus are moved from their current position however, the names will need to be updated within the template.
